.teamcity/components/settings.kt

```diff
@@ -119,6 +119,9 @@ var serviceTestConfigurationOverrides = mapOf(
         // Orbital is only available in certain locations
         "orbital" to testConfiguration(locationOverride = LocationConfiguration("eastus", "southcentralus", "westus2", false)),
 
+        // Network Regional Tire Public IP is only available in
+        "network" to testConfiguration(locationOverride = LocationConfiguration("westeurope", "eastus2", "westus", false)),
```

Comment:
Are we sure this won't break any other network tests that rely on those locations?

Reply:
I run the Network tests in TeamCity of this PR. but the location_alt2 is still `westus2`. isn't this how TC works?

![image](https://github.com/hashicorp/terraform-provider-azurerm/assets/2633022/37b44cc0-5412-48a3-af47-e723e8f81c17)

Reply:
Those 3 Regions should be fine

@wuxu92 the TeamCity configuration is only pulled from `main`, so this won't show up there until it's merged, fwiw